sympy/sets/fancysets.py

```diff
@@ -298,6 +298,16 @@ def TransformationSet(*args, **kwargs):
     return ImageSet(*args, **kwargs)
 
 
+class Complex(with_metaclass(Singleton, ImageSet)):
```

Comment:
Or perhaps ComplexPlane. (I believe the idea was to have a set for geometric purposes.)